application/config/config.go

```diff
@@ -209,6 +213,11 @@ func New() *Config {
 	c.deviceId = c.determineDeviceId()
 	c.addDefaults()
 	c.filterSeverity = lsp.DefaultSeverityFilter()
+	c.engine = app.CreateAppEngine()
```

Comment:
suggestion: the latest gaf changes should allow us to inject a logger via `CreateAppEngineWithLogger`, which may be useful for observability purposes